packages/grafana-data/src/types/logs.ts

```diff
@@ -33,6 +33,25 @@ export enum LogLevel {
   unknown = 'unknown',
 }
 
+// LogLevel: add syslog severity levels
+                          // syslog keyword:
+// @ts-ignore
```

Comment:
We should find a way to support this without having to `@ts-ignore` it.

Reply:
Aggred. I tried to do this without too much rewriting, but did't find a way.

Reply:
We can help with that. Will suggest next steps after an internal discussion. Thanks!